website_v2/gcloud-powershell-website-v2/www/static/app/mainController.js

```diff
@@ -0,0 +1,12 @@
+var app = angular.module('powershellSite');
+
+app.controller('MainController',
+    function($scope, $route, $routeParams, $location, $http) {
+    $scope.$route = $route;
+    $scope.$location = $location;
+    $scope.$routeParams = $routeParams;
+    $http.get('static/_data/cmdletsFull.json')
+       .then(function(res) {
+           $scope.products = res.data;
```

Comment:
I'm not opposed to leaving this on the scope.  It's just not the best practice.

+1 to default to null.